css-positioning/absolute.css

```diff
@@ -0,0 +1,46 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+
+  /* top: 20px;
+  left: 245px; */
+  top: 1rem;
+  right: -12rem;
+  width: 250px;
```

Comment:
Oh no! What happened here? You shouldn't need negatives values here, and you shouldn't need to set a width 🤔